dsms/knowledge/webform.py

```diff
@@ -566,6 +568,7 @@ def _validate_inputs(cls, self: "Entry") -> "Entry":
             default_value = spec.value
             select_options = spec.select_options
             range_options = spec.range_options
+            knowledge_type = spec.knowledge_type
```

Comment:
We should probably rename it to knowledge_types (plural)

Reply:
Do you mean in the SDK or the backend?

Reply:
I can introduce an alias in the sdk which will work with both versions, in plural or singular. The adaption in the backend can be done in the next platform release and the sdk will already be compatible with that.

Reply:
I don't know if I like the alias approach as I slightly prefer a more strict approach, but ok.

Reply:
It is definitely a very specific solution, which becomes obsolete once the backend is adapted. However, if we want to do it properly, we would need to make a patch for the platform in terms of the frontend _and_ backend first before adapt it here in the sdk. This will take much more time.